admin-ui/src/pages/request-detail-page.tsx

```
@@ -247,8 +247,23 @@ export function RequestDetailPage(): React.JSX.Element {
                     ) : (
                       ""
                     )}
+                    {item.affinity_hit ? (
```

ESLint

[error] Delete `·(`

(prettier/prettier)